server/task/src/test/java/org/apache/james/task/eventsourcing/EventSourcingTaskManagerTest.java

```diff
@@ -19,37 +19,44 @@
 
 package org.apache.james.task.eventsourcing;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import org.apache.james.eventsourcing.EventSourcingSystem;
 import org.apache.james.eventsourcing.eventstore.EventStore;
 import org.apache.james.eventsourcing.eventstore.memory.InMemoryEventStore;
 import org.apache.james.task.CountDownLatchExtension;
 import org.apache.james.task.MemoryWorkQueue;
 import org.apache.james.task.SerialTaskManagerWorker;
+import org.apache.james.task.Task;
+import org.apache.james.task.TaskId;
 import org.apache.james.task.TaskManager;
 import org.apache.james.task.TaskManagerContract;
 import org.apache.james.task.TaskManagerWorker;
 import org.apache.james.task.WorkQueue;
 
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import scala.Function1;
 
 @ExtendWith(CountDownLatchExtension.class)
 class EventSourcingTaskManagerTest implements TaskManagerContract {
 
+    private static final String HOSTNAME = Hostname.apply("foo");
```

Comment:
I can't get why the type here is a string and not `Hostname`...

Reply:
Scala value class don't be play nice when used in java. It as been reworked in a fixup. 
The Hostname class is now a simple scala case class. And the `HOSTNAME` const is now rightly typed.